src/main/java/org/motechproject/mots/errorhandling/GlobalErrorHandling.java

```diff
@@ -57,4 +58,11 @@ public ErrorResponse entityNotFoundException(EntityNotFoundException ex) {
   public String handleAccessDeniedException(Exception ex, WebRequest request) {
     return ex.getMessage();
   }
+
+  @ExceptionHandler(LocationException.class)
```

Comment:
I think you should rename this exception to something like this "MotsAccessDeniedException" or something similar, but indicating that the user don't have rights to access the resource